decidim-core/app/views/decidim/pages/index.html.erb

```diff
@@ -7,84 +7,68 @@ edit_link(
             <% if topic.pages.any? %>
-              <div class="column">
-                <div class="card card--process card--mini">
-                  <div class="card__content">
-                    <h3 class="card__title">
-                      <%= link_to translated_attribute(topic.title), page_path(topic.pages.first), class: "card__link" %>
+              <div class="page__accordion">
+                <div class="page__accordion-trigger">
+                  <div>
+                    <h3 id="accordion-title-<%= ix %>" class="page__accordion-trigger-title">
+                      <%= link_to translated_attribute(topic.title), page_path(topic.pages.first) %>
                     </h3>
-                    <p><%= translated_attribute topic.description %></p>
-                    <ul>
-                      <% topic.pages.each do |page| %>
-                        <li><%= link_to translated_attribute(page.title), page_path(page) %></li>
-                      <% end %>
-                    </ul>
-                  </div>
-                  <div class="card__footer">
-                    <div class="card__support">
-                      <%= link_to page_path(topic.pages.first),
-                                  class: "card__button button button--sc small" do %>
-                        <%= t(".read_more") %>
-                        <span class="show-for-sr"><%= translated_attribute(topic.title) %></span>
-                      <% end %>
-                    </div>
+                    <p class="page__accordion-trigger-snippet"><%= translated_attribute topic.description %></p>
                   </div>
+                  <button id="accordion-trigger-<%= ix %>" data-controls="accordion-panel-<%= ix %>" aria-labelledby="accordion-title-<%= ix %>">
+                    <%= icon "arrow-down-s-line", class: "page__accordion-trigger-arrow" %>
+                    <%= icon "arrow-up-s-line", class: "page__accordion-trigger-arrow" %>
+                  </button>
+                </div>
+
+                <div id="accordion-panel-<%= ix %>" class="page__accordion-panel">
+                  <ul role="menu">
+                    <% topic.pages.each do |page| %>
+                      <li role="menuitem"><%= link_to translated_attribute(page.title), page_path(page), class: "text-secondary" %></li>
+                    <% end %>
+                  </ul>
                 </div>
               </div>
             <% end %>
           <% end %>
         </div>
       </div>
-    </div>
-  <% end %>
+    <% end %>
 
-  <% if @orphan_pages.any? %>
-    <div class="row">
-      <h3 class="section-heading"><%= t ".standalone_pages" %></h3>
-      <div class="row collapse">
-        <div class="row small-up-1 smallmedium-up-2 mediumlarge-up-2 large-up-3 card-grid">
+    <% if @orphan_pages.any? %>
+      <div class="space-y-6">
+        <h2 class="h2 decorator"><%= t "decidim.pages.index.standalone_pages" %></h2>
+        <div class="grid grid-cols-2 gap-6">
```

Comment:
This looks somewhat cluttered on mobile screens, should it be 100% width maybe?

![Cluttered card on mobile](https://user-images.githubusercontent.com/864340/183087584-b8d7a17e-48b3-4d52-807f-866a8efc01fd.png)